packages/core/src/telemetry/metrics.ts

```diff
@@ -0,0 +1,278 @@
+import { createStore } from '@segment/sovran-react-native';
+import { checkResponseForErrors } from '../errors';
+import { allSettled, chunk } from '../util';
+
+/**
+ * Limit number of tags accepted by the server
+ */
+const TAGS_LIMIT = 10;
+
+/**
+ * We prioritize these tags over user defined ones if the number is over the limit, ordered by importance
+ */
+const PRIORITED_TAGS = [
```

Comment:
should this be `PRIORITIZED_TAGS`?

Reply:
Yup, must've been editing this way too late